packages/gateway/plugins/auth.ts

```
@@ -72,42 +74,38 @@
     case 'ACCOUNTANT':
+      return ['ACCOUNTANT'];
+    case 'ADMIN':
       return ['ACCOUNTANT', 'ADMIN'];
     default:
       return [];
   }
-};
+}
 
-const validateUser: ValidateUserFn<UserType> = ({ user, fieldDirectives, parentType }) => {
+export function validateUser({ user, fieldDirectives, parentType }: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)